examples/http/pfsense_pfinfo.js

```diff
@@ -0,0 +1,493 @@
+        }
+        d.resolve(body);
+    });
+    return d.promise;
+
+}
+
+
+/**
+ * 
+ * @param {*} init object initialisation
+ * @param {*} object object to clone
+ * @returns cloned object
+ */
+function clone(init, object) {
+    var toReturn = JSON.parse(JSON.stringify(object));
+    Object.keys(init).forEach(function (key) {
+        toReturn[key] = init[key];
+    });
+    return toReturn;
+}
+
+function trim(s) {
+    return s.trim().replace(/\s+/g, " ");
+}
+
+/**
+ * 
+ * @param {*} body pfinfo page
+ * @param {*} paramName parameter name to extract from pfinfo page
+ * @returns the value of the parameter and it's unit
+ */
+function extractParameterValue(body, paramName) {
+    var regex = new RegExp(paramName + "\\s+([^\n]*)\n");
+    var match = body.match(regex);
+    var value = "";
+    if (match) value = match[1];
+    if (value) {
+        var valueUnit = value.split(" ");
+        if (valueUnit.length == 2) {
+            return { value: valueUnit[0], unit: valueUnit[1] };
+        }
+        if (value.indexOf("s") >= 0) return { value: value.substring(0, value.length - 1), unit: "s" };
+        return { value: value };
+    }
+}
+
+
+
+/**
+ * This function extracts variables from the pfinfo page
+ * @param {string} body 
+ */
+function extract_variables(body) {
+    // extract em1 interface stats
+    var match = body.match(em1StatsRegex);
+    if (match) {
+        _vars.push(_var("em1_bytes_in_ipv4", "em1 Bytes In ipv4", match[1]));
+        _vars.push(_var("em1_bytes_out_ipv4", "em1 Bytes Out ipv4", match[3]));
+        _vars.push(_var("em1_packets_in_passed_ipv4", "em1 packets in passed ipv4", match[5]));
+        _vars.push(_var("em1_packets_in_blocked_ipv4", "em1 packets in blocked ipv4", match[7]));
+        _vars.push(_var("em1_packets_out_passed_ipv4", "em1 packets out passed ipv4", match[9]));
+        _vars.push(_var("em1_packets_out_blocked_ipv4", "em1 packets out blocked ipv4", match[11]));
+        _vars.push(_var("em1_bytes_in_ipv6", "em1 Bytes In ipv6", match[2]));
+        _vars.push(_var("em1_bytes_out_ipv6", "em1 Bytes Out ipv6", match[4]));
+        _vars.push(_var("em1_packets_in_passed_ipv6", "em1 packets in passed ipv6", match[6]));
+        _vars.push(_var("em1_packets_in_blocked_ipv6", "em1 packets in blocked ipv6", match[8]));
+        _vars.push(_var("em1_packets_out_passed_ipv6", "em1 packets out passed ipv6", match[10]));
+        _vars.push(_var("em1_packets_out_blocked_ipv6", "em1 packets out blocked ipv6", match[12]));
+    }
+
+    // extract totalRateParams variables
+    totalRateParams.forEach(function (p) {
+        var regex = totalRateStatsRegex(p.title, p.params);
+        match = body.match(regex);
+        if (match) {
+            for (var i = 1; i < match.length; i += 2) {
+                var j = (i - 1) / 2;
+                _vars.push(_var(p.title + "_" + p.params[j] + "_total", p.title + ": " + p.params[j] + " (total)", match[i]));
```

Comment:
Since 'table' is a reserved word can you change with the following:

`                var variableIdPartial = p.title + "_" + p.params[j]
                variableIdPartial = variableIdPartial.replace('table', 'tbl') // table is a reserved word
                _vars.push(_var(variableIdPartial + "_total", p.title + ": " + p.params[j] + " (total)", match[i]));
                _vars.push(_var(variableIdPartial + "_rate", p.title + ": " + p.params[j] + " (rate)", match[i + 1].substring(0, match[i + 1].length - 2), "s"));`